taskcluster/tc-build-utils.sh

```diff
@@ -337,6 +410,116 @@ do_nuget_build()
   nuget pack nupkg/deepspeech.nuspec
 }
 
+do_nuget_repackage()
+{
+  package_option=$1
+  PROJECT_NAME=$2
+  
+  if [ -z "${PROJECT_NAME}" ]; then
+    exit "Please call with a valid PROJECT_NAME"
+    exit 1
+  fi;
+
+  
+  # Forming nuget name
+  nuget="${PROJECT_NAME}.${DS_VERSION}.nupkg"
+
+  # Getting Nuget artifact URL
+  nuget_pkg_url=$(get_dep_nuget_pkg_url "${nuget}")
+    
+  cd ${DS_DSDIR}/native_client/dotnet
+  ${WGET} -O - "${nuget_pkg_url}" 
+  
+  cd ${DS_DSDIR}/native_client/dotnet/nupkg
+  gunzip > "${DS_DSDIR}/native_client/dotnet/${nuget}"
+  
+  local all_deps="$(curl -s https://community-tc.services.mozilla.com/api/queue/v1/task/${TASK_ID} | python -c 'import json; import sys; print(" ".join(json.loads(sys.stdin.read())["dependencies"]));')"
```

Comment:
please extract that into a function later for fetching the deps

Reply:
see `get_dep_nuget_pkg_url`, either augment it for fetching ALL urls or add a second function doing it, in `tc-dotnet-utils.sh`